lib/std/posix.zig

```diff
@@ -3522,6 +3533,9 @@ pub fn isatty(handle: fd_t) bool {
             }
         }
     }
+    if (native_os == .uefi) {
+        return false;
```

Comment:
Why doesn't this use the `isatty` in `uefi.posix`?

Reply:
Didn't realize that existed when I was comparing the errors `zig build -Dtarget=aarch64-uefi` was giving vs the diff. Will fix.

Reply:
This is now using `uefi.posix`.